test/connect_params.py

```diff
@@ -0,0 +1,118 @@
+import yaml
```

Comment:
include this file in test/kafka_connect_data_enrichment_tests folder as it specific to that test